charts/refinery/values.yaml

```diff
@@ -66,6 +66,8 @@ extraVolumes: [ ]
 
 # Values used to build config.yaml.
 # See the example in sample-configs/config_complete.yaml for full details on all properties
+# Supports templating. To escape existing instances of {{ }}, use {{` <original content> `}}.
+# For example, {{ REDACTED_EMAIL }} becomes {{` {{ REDACTED }} `}}.
```

Comment:
```suggestion
# For example, {{ REDACTED_EMAIL }} becomes {{` {{ REDACTED_EMAIL }} `}}.
```

I think the example of escaping is just adding backticks, right?

Reply:
Extra {{ }} as well, but the rename was unintended.